hub/management/commands/import_council_carbon_neutral_data.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.import_utils import filter_authority_type
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+declare_map = {
+    "Y": "Yes",
+    "N": "No",
+}
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss_code"
+    message = "Importing council net zero target data"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "label": "Net Zero Date",
```

Comment:
I’d probably word this as “Net Zero target date”.

Reply:
Or even… “Council Net Zero target date”, to make the scope clearer when you’re looking at this dataset on the `/explore` page?